src/Exception/ObjectGeneratorPersisterException.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Fidry\AliceDataFixtures package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\AliceDataFixtures\Exception;
+
+use LogicException;
+
+/**
+ * @private
+ */
+final class ObjectGeneratorPersisterException extends LogicException
+{
+    /**
+     * No ID found for the entity "EntityName". If this entity has an auto ID generator, 
```

Comment:
no sorry I meant as the error message instead of the one bellow :)

I also forgot, instead of making this an exception, I think we should make it a factory. So we could rename this to `*Factory`, and this `entityMissingAssignedIdForField($entity)` method could become `createForEntityMissingAssignedIdForField($entity): LogicException`.

We don't need a custom exception: nobody should catch this one as it requires a manual change from the user